curvefs/src/client/dentry_cache_manager.cpp

```diff
@@ -156,5 +156,26 @@ CURVEFS_ERROR DentryCacheManagerImpl::ListDentry(uint64_t parent,
     return CURVEFS_ERROR::OK;
 }
 
+CURVEFS_ERROR DentryCacheManagerImpl::CheckDirEmpty(uint64_t parent,
+                                            std::list<Dentry> *dentryList) {
```

Comment:
`indentation `

Reply:
the second line is larger than 80 characters?